HW_8/task_3_to_upper_with_map.py

```diff
@@ -0,0 +1,17 @@
+# Task 3. (optional) Convert all list elements of type string to upper case using map.
+
+init_lst = [1, 'hello', 'word', True]
+
+
+def to_uppercase(elem):
+    return elem.upper() if type(elem) == str else elem
```

Comment:
nit: nice function for checking type is `isinstance()`

Reply:
Ok